### DISCUSSION TITLE
SMplayer subtitles and gui font problem

### DISCUSSION THREAD

**Discussion by spamator12** [smplayer-dev/smplayer#1051] [General]
It looks like here.

https://imgur.com/a/oBbkIhK

Also SMplayer cannot open some movie I have downloaded from youtube (thanks WONDERFUL Video DownloadHelper - https://addons.mozilla.org/pl/firefox/addon/video-downloadhelper/).

I hope it can be corrected (or fixed in a new program version) - let me know how.

**Comment by smplayer-dev** [comment]
You can change the font of the subtitles in preferences -> subtitles.

**Comment by spamator12** [comment]
I have already tried this - it is not working. The interface -> subtitles or interface -> interface -> default font are not helpful in this scenario.

**Comment by smplayer-dev** [comment]
What's the format of the subtitles?

**Comment by spamator12** [comment]
Hi, sorry for the late reply, for example - Cobweb (2023) (1080p BluRay x265 10bit Tigole) - those are subtitle inside the movie - how to check it?

Th other problem however is more serious. I don't know why SMplayer cannot run some of the files download thanks to Video DownloadHelper - https://addons.mozilla.org/pl/firefox/addon/video-downloadhelper/ probably some codecs missing, but I don;t know how to fix this or what codes those are.

**Comment by spamator12** [comment]
Demuxer: lavfpref


Initial Audio Stream
Format: MP4A
Bitrate: 256 kbps
Rate: 48000 Hz
Channels: 2
Selected codec: ffaac
